README.md

```diff
@@ -1,30 +1,37 @@
 # Malware Response Using Symantec Solutions
 
+## Release Information
+- Solution Pack Version: 1.0.0
+- Minimum Compatible FortiSOAR™ Version : 7.2.0
+- Authored By: Fortinet
+- Certified: No
+
 ## Overview
 ### Introduction
-This article describes the  Malware Response Using Symantec Solutions Pack. The solution pack demonstrates the scenarios and use cases around Malware resported by Symantec EDR.
-
-The scenario demonstrates and generates a demo alert for the Alert Type 'Malware' reported by Symantec EDR.
+*Malware Response Using Symantec Solutions* is designed to provide a set of investigation playbooks to respond to the Malware alert provided by Symantec EDR.
 
-The usecase deals with containment and investigation procedures once the Symantec EDR reports a Malware Alert.
+Configure Malaware ingestion using Symantec EDR connector. Ingestion process creates an alert of type 'Malware', and then triggers the response workflow.
```

Comment:
Malware ingestion?

Reply:
who triggers the response workflow? manual or automatic?

Reply:
trigger means manually

Reply:
Malware ingestion -> means malware Alert

Reply:
Configure Malaware ingestion using Symantec EDR connector --> Configure ingestion using Symantec EDR connector
triggers the response workflow. --> manually trigger the....